### DISCUSSION TITLE
Problemas de compilación

### DISCUSSION THREAD

**Discussion by Clirussa** [INGCOM-UNRN-P1/cursada-2023#71] [Preguntas y respuestas] [answered]
Hola, no puedo compilar, no entiendo cual es el problema.  Esto intentando con el ejercicio 1. 

![Error de compilación](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/102642611/ef369234-3153-4b10-a22b-5f0ed2dab6b4)

**Comment by camilaparedesrobles** [comment, marked as answer]
Vicky, sabés que me pasó lo mismo porque tenía el repositorio alojado, como vos, en las carpetas con el material de la cátedra. Los títulos de las carpetas, que tienen tildes básicamente, complican todo. Yo alojé el repositorio directamente en el escritorio y ahí compiló sin problema.
Fijate si te sirve.

**Reply by Clirussa** [reply]
Dale gracias, ahora pruebo.

**Reply by martinvilu** [reply]
Te agrego también, espacios, la ubicación entre más simple, mejor
